f8a_worker/workers/githuber.py

```diff
@@ -51,14 +51,15 @@ def _get_last_years_commits(self, repo_url):
     def _get_repo_stats(self, repo):
         """Collect various repository properties."""
         try:
-            if repo.get('contributors_url', ''):
-                contributors = get_response(repo.get('contributors_url', ''), self._headers)
+            url = repo.get('contributors_url', '')
+            if url:
+                contributors = get_gh_contributors(url)
             else:
-                contributors = {}
+                contributors = 0
```

Comment:
It is impossible to have a zero contributor for a repo, incase of unforeseen errors, -1 would be the right choice I guess.